README.md

```diff
@@ -21,9 +21,11 @@ $ gem install lrama
 From source codes,
 
 ```shell
+$ cd "$(lrama root)"
+$ bundle install
 $ bundle exec rake install
-$ lrama --version
-0.1.0
+$ bundle exec lrama --version
```

Comment:
@jinroq  Just confirmation, `lrama --version` does not work well? I guess `bundle exec rake install` install lrama executable outside of bundler control, so it might work without bundler command. Can I ask you to check `lrama --version`?

Reply:
@yui-knk 

```
$ lrama --version
コマンド 'lrama' が見つかりません。もしかして:
  command 'llama' from snap llama (1.4.0)
他のバージョンについては 'snap info <snapname>' を確認してください。
```

System requirements: Ubuntu 22.04.2 LTS (Jammy Jellyfish)

Should I fix rake's settings so that `lrama --version` can execute?